[WIP] Prague 2020 full website (#1180)
writethedocs/www: docs/include/conf/prague/visiting.rst

Where to stay
-------------

Hotel Olšanka is located in Prague 3, Žižkov district. This neighborhood is close to the city center, and there are plenty of accommodation options in the area for any budget. If you want to stay at the hotel, we have secured a discount code for our attendees. 

> Please note that the hotel does not offer accessible rooms, and Write the Docs cannot be held liable for mobility issues outside the conference areas, which are fully accessible.

Reviewer: ```suggestion
Please note that the hotel does not offer accessible rooms, and Write the Docs is not responsible for mobility issues outside the conference areas, which are fully accessible.
```
Slightly less legalese.
